tests/integration/pkg/tests/custom/custom_test.go

```diff
@@ -11,8 +12,25 @@ import (
 	"github.com/rancher/rancher/tests/integration/pkg/systemdnode"
 	"github.com/stretchr/testify/assert"
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func TestSystemAgentVersion(t *testing.T) {
+	clients, err := clients.New()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer clients.Close()
+
+	setting, err := clients.Mgmt.Setting().Get("system-agent-version", metav1.GetOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	assert.False(t, setting.Value == "")
```

Comment:
You could use https://pkg.go.dev/github.com/stretchr/testify/assert#NotEmpty, however, the check below covers this as well I believe. Unless we're trying to account for case when both `os.Getenv("CATTLE_SYSTEM_AGENT_VERSION")` and `setting.Value` end up empty and it should fail the test.

Reply:
Yes, just want to make sure both things are not empty.